images/baremetal-runtimecfg.yml

```diff
@@ -21,21 +21,9 @@ enabled_repos:
 for_payload: true
 from:
   builder:
-  - stream: rhel-9-golang-1.20
+  - stream: rhel-9-golang
```

Comment:
Maybe it makes sense to bump the golang builder in a different PR? Should any trouble arise, we'd just need to revert a short PR